srv/salt/ceph/ganesha/files/silver-common.conf.j2

```diff
@@ -6,9 +6,9 @@ EXPORT
 {
 	Export_ID={{ loop.index }};
 
-	Path = "/";
+	Path = "demo";
```

Comment:
I'd suggest creating a bucket as `{{user}}-demo` or something, this is because bucket names are unique in S3 by default and this would lead to clashes for the second user